build.mjs

```diff
@@ -0,0 +1,108 @@
+import esbuild from "esbuild";
+import { glob } from "glob";
+import fs from 'fs';
+import ts from 'typescript';
+
+const components = glob.sync("packages/**/src/index.ts");
+const indexPath = "packages/index.ts";
```

Comment:
Should we expose dead-toggle, clickable, affix and so on from _helpers/ folder as separate components, too?

Reply:
Good point, didn't know that 👍

Reply:
These helper components needs to be moved out of _helpers and should follow the same pattern (packages/**/src) for typescript to work properly. I'll create a new task for that and do those in the next PR.